tracing-appender/src/rolling.rs

```diff
@@ -97,7 +101,10 @@ pub struct RollingFileAppender {
 /// [writer]: std::io::Write
 /// [`MakeWriter`]: tracing_subscriber::fmt::writer::MakeWriter
 #[derive(Debug)]
-pub struct RollingWriter<'a>(RwLockReadGuard<'a, File>);
+pub struct RollingWriter<'a> {
+    inner: RwLockReadGuard<'a, File>,
+    current_size: Arc<AtomicU64>,
```

Comment:
@CBenoit, thank you very much for the feature. `RollingWriter` borrows `RollingFileAppender`, looks like there might be `current_size: &'a AtomicU64` instead of Arc. It allows also us to avoid Arc in Inner.

Reply:
You’re probably right, I’ll try that!